apps/nova/src/components/Settings/TopicSelector.tsx

```diff
@@ -0,0 +1,91 @@
+import { useFormatMessage } from "@litespace/luna/hooks/intl";
+import { MultiSelect } from "@litespace/luna/MultiSelect";
+import { Typography } from "@litespace/luna/Typography";
+import React, { useCallback, useEffect, useMemo, useState } from "react";
+import { Loader } from "@litespace/luna/Loading";
+import { useTopics, useUserTopics } from "@litespace/headless/topic";
+import { isEqual } from "lodash";
+import { TopicWatcher } from "@/components/Settings/ProfileForm";
+
+export const TopicSelector: React.FC<{
+  setSubmitFlag: (props: { userData: boolean; topics: boolean }) => void;
+  submitFlag: { userData: boolean; topics: boolean };
+  setNewTopics: React.Dispatch<React.SetStateAction<TopicWatcher>>;
+}> = ({ setSubmitFlag, submitFlag, setNewTopics }) => {
+  const intl = useFormatMessage();
+  const [topics, setTopics] = useState<number[]>([]);
+  const allTopicsQuery = useTopics({});
+  const userTopicsQuery = useUserTopics();
+
+  const allTopics = useMemo(() => {
+    if (!allTopicsQuery.query.data?.list) return [];
+    return allTopicsQuery.query.data.list.map((topic) => ({
+      value: topic.id,
+      label: topic.name.en,
```

Comment:
We should show the Arabic version of the topic. 


```suggestion
      label: topic.name.ar,
```